website/docs/r/app_service_certificate.html.markdown

```diff
@@ -60,6 +62,10 @@ data "azuread_service_principal" "MicrosoftWebApp" {
 }
 ```
 
+* `key_vault_id` - (Optional) The ID of the Key Vault. Must be specified if Key Vault of `key_vault_secret_id` is in different subscription from the App Service Certificate. Changing this forces a new resource to be created.
+
+-> **NOTE:** `key_vault_id` must be specified with `key_vault_secret_id`.
```

Comment:
This shouldn't be, and doesn't need to be a requirement
```suggestion
```

Reply:
changed to 
```
-> **NOTE:** `key_vault_id` can only be specified if `key_vault_secret_id` has been set.
```